src/components/CategoryNavContainer/CategoryNav/CategoryNav.scss

```diff
@@ -1,12 +1,99 @@
 @import '~style/variables';
 
+.categoryNav {
+  background-color: $color-white;
+  display: flex;
+  height: 48px;
+  align-items: center;
+  padding-left: 20px;
+  width: 100%;
+  white-space: nowrap;
+  overflow-x: auto;
+  -webkit-overflow-scrolling: touch;
+  -ms-overflow-style: -ms-autohiding-scrollbar;
+
+  @include for-desktop-up {
+    height: 80px;
+    padding-left: 30px;
+  }
+}
+
 .link {
   display: inline-block;
-  padding: 10px;
+  padding: 0 20px;
   text-decoration: none;
+  height: 23px;
+  border-radius: 100px;
+  font-size: 12px;
+  margin-right: 10px;
+  color: $color-grey--700;
+  box-shadow: 0 0 4px 2px rgba(0, 0, 0, 0.1);
+
+  @include for-desktop-up {
+    font-size: 16px;
+    height: 27px;
+    box-shadow: initial;
+
+    &:hover {
+      color: $color-white;
+      box-shadow: 0 0 4px 2px rgba(0, 0, 0, 0.1);
```

Comment:
Does it make sense to centralise this? You can do a mixin for shadows of different sizes for consistency. I'm not sure if we have any/many other shadows though.